src/extensions/bigbang/test/bigbang_test.go

```diff
@@ -60,77 +60,79 @@ func TestReleases(t *testing.T) {
 		zarfCache = fmt.Sprintf("--zarf-cache=%s", CIMount)
 	}
 
+	ctx := context.Background()
```

Comment:
Use context.TODO() in tests.

Reply:
I originally was using `context.TODO()` in the tests, but @schristoff-du asked if we should use a real context in the tests, and I wasn't sure of when you should or shouldn't use TODO, so switched back to using `Background()`.

Will switch back to using TODO in tests.

Reply:
TODO is supposed to be a placeholder, in tests we can make a context and pass it through the functions to ensure we are tests the functions as close as they'll be used normally. In tests I have seen it more common to make a context, or pass a background.

Reply:
[Reference to source with notes on when to use Background vs TODO](https://github.com/golang/go/blob/master/src/context/context.go#L207)

Reply:
@schristoff @phillebaba updated to use `context.Background()` in tests again. I don't think it really matters what which one we use in the tests. The docs for `Background` explicitly states that it's typically used in tests so that is likely the best option of the two.